src/default_parameters.jl

```diff
@@ -8,8 +8,8 @@
 
     # DOMAIN RESOLUTION AND RATIO
     nx::Int=100                         # number of grid cells in x-direction
-    Lx::Real=4000e3                     # length of the domain in x-direction [m]
-    L_ratio::Real=2                     # Domain aspect ratio of Lx/Ly
+    Lx::Int=4000e3                      # length of the domain in x-direction [m]
+    L_ratio::Int=2                      # Domain aspect ratio of Lx/Ly
```

Comment:
what about `Float64`? While for `Lx` integer is probably fine, `L_ratio = 1.5` does make sense to me and should be allowed?